pyccel/codegen/printing/fcode.py

```diff
@@ -1310,10 +1312,42 @@ def _print_PythonImag(self, expr):
     #========================== List Methods ===============================#
 
     def _print_ListAppend(self, expr):
-        target = expr.list_obj
+        target = self._print(expr.list_obj)
         arg = self._print(expr.args[0])
         return f'call {target} % push_back({arg})\n'
 
+    def _print_ListPop(self, expr):
+        list_obj = expr.list_obj
+        target = self._print(list_obj)
+        index_element = expr.index_element
+        if isinstance(expr.current_user_node, Assign):
```

Comment:
the following works in C:
```python
if __name__ == '__main__':
    a = [1,2,3]
    print(a.pop(a.pop(0)))
```
Do you think you can support it here as well?

Reply:
Test added : a3d542c66
Fix pushed : 742ab8115